dev/release/verify-release-candidate.sh

```diff
@@ -51,11 +51,17 @@ HERE=$(cd `dirname "${BASH_SOURCE[0]:-$0}"` && pwd)
 
 ARROW_DIST_URL='https://dist.apache.org/repos/dist/dev/arrow'
 
-: ${ARROW_HAVE_CUDA:=}
```

Comment:
Initializing the default value is better than using multiple `${ARROW_CUDA:-OFF}`.